### PR TITLE
config: add new locksmithd options

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

**Comment by mischief** [comment]
fixes #392
fixes coreos/bugs#832

File: config/locksmith.go [hunk 1]
```diff
@@ -15,8 +15,11 @@
...
-	EtcdCAFile   string `yaml:"etcd_cafile"   env:"LOCKSMITHD_ETCD_CAFILE"`
-	EtcdCertFile string `yaml:"etcd_certfile" env:"LOCKSMITHD_ETCD_CERTFILE"`
-	EtcdKeyFile  string `yaml:"etcd_keyfile"  env:"LOCKSMITHD_ETCD_KEYFILE"`
+	Endpoint          string `yaml:"endpoint"      env:"LOCKSMITHD_ENDPOINT"`
+	EtcdCAFile        string `yaml:"etcd_cafile"   env:"LOCKSMITHD_ETCD_CAFILE"`
+	EtcdCertFile      string `yaml:"etcd_certfile" env:"LOCKSMITHD_ETCD_CERTFILE"`
+	EtcdKeyFile       string `yaml:"etcd_keyfile"  env:"LOCKSMITHD_ETCD_KEYFILE"`
+	Group             string `yaml:"group"	       env:"LOCKSMITHD_GROUP"`
+	RebootWindowStart string `yaml:"window_start"  env:"REBOOT_WINDOW_START"`
+	RebootWindowEnd   string `yaml:"window_end"    env:"REBOOT_WINDOW_START"`
```
Comment (jonboulle) [inline config/locksmith.go]: copypasta START

Comment (mischief) [inline config/locksmith.go]: caught me. :(

File: config/locksmith.go [hunk 2]
```diff
@@ -15,8 +15,11 @@
...
-	Endpoint     string `yaml:"endpoint"      env:"LOCKSMITHD_ENDPOINT"`
-	EtcdCAFile   string `yaml:"etcd_cafile"   env:"LOCKSMITHD_ETCD_CAFILE"`
-	EtcdCertFile string `yaml:"etcd_certfile" env:"LOCKSMITHD_ETCD_CERTFILE"`
-	EtcdKeyFile  string `yaml:"etcd_keyfile"  env:"LOCKSMITHD_ETCD_KEYFILE"`
+	Endpoint          string `yaml:"endpoint"      env:"LOCKSMITHD_ENDPOINT"`
+	EtcdCAFile        string `yaml:"etcd_cafile"   env:"LOCKSMITHD_ETCD_CAFILE"`
+	EtcdCertFile      string `yaml:"etcd_certfile" env:"LOCKSMITHD_ETCD_CERTFILE"`
+	EtcdKeyFile       string `yaml:"etcd_keyfile"  env:"LOCKSMITHD_ETCD_KEYFILE"`
+	Group             string `yaml:"group"	       env:"LOCKSMITHD_GROUP"`
+	RebootWindowStart string `yaml:"window_start"  env:"REBOOT_WINDOW_START"`
```
Comment (crawford) [inline config/locksmith.go]: Bonus points if you can provide a regular expression for validation.

File: config/locksmith.go [hunk 3]
```diff
@@ -15,8 +15,11 @@
...
+	RebootWindowEnd   string `yaml:"window_end"    env:"REBOOT_WINDOW_END"`
```
Comment (crawford) [inline config/locksmith.go]: REBOOT_WINDOW_LENGTH

**Comment by crawford** [comment]
fixes https://github.com/coreos/bugs/issues/818

**Comment by crawford** [comment]
Superseded by https://github.com/coreos/coreos-cloudinit/pull/395.

**Pull request closed without merging**